.github/CODEOWNERS

```diff
@@ -130,13 +130,13 @@
 
 # PHP
 /lib                                                                 @timothybjacobs @spacedmonkey
-/lib/global-styles.php                                               @timothybjabocs @spacedmonkey @oandregal
+/lib/global-styles.php                                               @timothybjacobs @spacedmonkey @oandregal
 /lib/class-wp-rest-block-editor-settings-controller.php              @timothybjacobs @spacedmonkey @geriux @antonis
-/lib/compat/wordpress-5.9/theme.json                                                      @timothybjabocs @spacedmonkey @oandregal
-/lib/compat/wordpress-5.9/theme-i18n.json                                                 @timothybjabocs @spacedmonkey @oandregal
-/lib/compat/wordpress-5.9/class-wp-theme-json-gutenberg.php          @timothybjabocs @spacedmonkey @oandregal
-/lib/compat/wordpress-5.9/class-wp-theme-json-resolver-gutenberg.php @timothybjabocs @spacedmonkey @oandregal
-/lib/full-site-editing                                               @janw-me
+/lib/compat/wordpress-5.9/theme.json                                 @timothybjacobs @spacedmonkey @oandregal
+/lib/compat/wordpress-5.9/theme-i18n.json                            @timothybjacobs @spacedmonkey @oandregal
+/lib/compat/wordpress-5.9/class-wp-theme-json-gutenberg.php          @timothybjacobs @spacedmonkey @oandregal
+/lib/compat/wordpress-5.9/class-wp-theme-json-resolver-gutenberg.php @timothybjacobs @spacedmonkey @oandregal
+/lib/full-site-editing
```

Comment:
We probably can remove this line since it contains now only a single file. In the past, we had there many files but it's now part of the WordPress core.